src/RamlToOpenApiConverter/RamlConverter.Components.cs

```diff
@@ -132,9 +132,12 @@ private OpenApiSchema MapSchema(IDictionary<object, object> properties, ICollect
                 }
                 else if (propertyType != null && _types.ContainsKey(propertyType))
                 {
-                    // Simple Type
                     var simpleType = _types.GetAsDictionary(propertyType);
-                    schema = MapParameterOrPropertyDetailsToSchema(simpleType);
+                    var props = simpleType?.GetAsDictionary("properties");

```

Comment:
rewrite as:
``` c#
var properties = simpleType?.GetAsDictionary("properties");

schema = properties != null ?
    CreateDummyOpenApiReferenceSchema(propertyType) : // Custom type
    MapParameterOrPropertyDetailsToSchema(simpleType); //  Simple Type
```

Reply:
@StefH I already made your mentioned changes in v1.1 commit (please ignore my v.10 commit)
thank you @StefH  (:

Reply:
Can you indent the 2 lines by 4 spaces?
So that it looks like my comment above?